pkg/option/config.go

```diff
@@ -2170,6 +2180,16 @@ func (c *DaemonConfig) Validate() error {
 		return fmt.Errorf("specified PolicyMap max entries %d must not exceed maximum %d",
 			c.PolicyMapMaxEntries, policyMapMax)
 	}
+	fragmentsMapMin := (1 << 8)
+	fragmentsMapMax := (1 << 16)
+	if c.FragmentsMapEntries < fragmentsMapMin {
+		return fmt.Errorf("specified fragments tracking map max entries %d must exceed minimum %d",
```

Comment:
```suggestion
		return fmt.Errorf("specified max. entries for fragment-tracking map %d must exceed minimum %d",
```
`max.` is an abbreviation so needs a dot; `fragment-tracking` is a compound adjective so needs an en dash and singular form.